lib/main.dart

```diff
@@ -9,11 +13,66 @@ import 'package:seeds/screens/app/app.dart';
 import 'package:seeds/screens/onboarding/onboarding.dart';
 import 'package:seeds/widgets/passcode.dart';
 import 'package:seeds/widgets/splash_screen.dart';
+import 'package:sentry/sentry.dart' as Sentry;
 
 import 'generated/r.dart';
 
+final Sentry.SentryClient _sentry = Sentry.SentryClient(
+    dsn: "https://ee2dd9f706974248b5b4a10850586d94@sentry.io/2239437");
+
+bool get isInDebugMode {
+  bool inDebugMode = false;
+  assert(inDebugMode = true);
```

Comment:
Here operation of assignment is used (instead of comparison)
That means you assign inDebugMode equal to true in this line (and equal to false in previous line)
Oh, I see the idea... In production mode assert statements are ignored and therefore function will return false, but in debug mode it will return true

Reply:
Yes, assert only works in debug mode.

Reply:
:+1: